security/org-policies/policies/RestrictivePolicy.json

```diff
@@ -283,6 +283,69 @@
                     ]
                 }
             }
-        }
+        },
+        {
+            "Sid": "DenyAWSBackuponSpecificTag",
+            "Effect": "Deny",
+            "Action": [
+              "backup:CreateBackupPlan",
+              "backup:CreateBackupSelection",
+              "backup:CreateBackupVault",
+              "backup:CreateFramework",
+              "backup:CreateLegalHold",
+              "backup:CreateReportPlan",
+              "backup:DeleteBackupPlan",
+              "backup:DeleteBackupSelection",
+              "backup:DeleteBackupVault",
+              "backup:DeleteBackupVaultLockConfiguration",
+              "backup:DeleteBackupVaultNotifications",
+              "backup:DeleteFramework",
+              "backup:DeleteRecoveryPoint",
+              "backup:DeleteReportPlan",
+              "backup:DeleteBackupVaultAccessPolicy",
+              "backup:DisassociateRecoveryPoint",
+              "backup:DisassociateRecoveryPointFromParent",
+              "backup:PutBackupVaultAccessPolicy",
+              "backup:UpdateBackupPlan",
+              "backup:UpdateFramework",
+              "backup:UpdateRecoveryPointLifecycle",
+              "backup:UpdateReportPlan"
+            ],
+            "Resource": "*",
+            "Condition": {
+              "StringEquals": {
+                "aws:ResourceTag/dfds.owner": [
+                  "itbuildsourcedevex@dfds.com"
```

Comment:
Keeping an email address in a git repo is sub-optimal, but since this is a JSON file we would need some refactoring first. 

I will approve this PR as is, but would you be so kind to create an issue on the Ops board about refactoring SCP from JSON files to terraform objects?

Reply:
Thanks for the heads up @avnes 
Will try to replace the value with a variable instead rather than merge the PR. Pushing another commit once it's done.